qiskit/providers/__init__.py

```diff
@@ -488,10 +488,10 @@ def has_realtime_logic(circuit: QuantumCircuit) -> bool:
 handles submitting the circuits to the backend to be executed and returning a
 :class:`~qiskit.providers.Job` object. Depending on the type of backend this
 typically involves serializing the circuit object into the API format used by a
-backend. For example, on IBM backends from the ``qiskit-ibm-provider``
-package this involves converting from a quantum circuit and options into a
-:mod:`.qpy` payload embedded in JSON and submitting that to the IBM Quantum
-API. Since every backend interface is different (and in the case of the local
+backend. For example, a provider service might need to convert from a quantum
+circuit and options into a
+:mod:`.qpy` payload embedded in JSON before submitting it to a REST API.
```

Comment:
This piece of doing QPY embedded in JSON is very specific to how IBM does it. I'm not aware of any other provider that would do it this way, and if we were starting from scratch I don't think we would do this. It was supposed to be a specific example. I'd say we should say this is for the IBM provider either in the abstract or just update it to say qiskit-ibm-runtime. Or alternatively just remove the sentence.

Reply:
Just remove it in 5f0c52c8c6d5625c473e1a6e7c9d5eddbbe15850